wall_dist cmd
---
wall_dist resp
bool accepted